exchanges/exchange_test.go

```diff
@@ -492,7 +494,7 @@ func TestLoadConfigPairs(t *testing.T) {
 			},
 			Pairs: map[asset.Item]*currency.PairStore{
 				asset.Spot: {
-					RequestFormat: &currency.PairFormat{},
+					RequestFormat: &currency.PairFormat{}, // NOTE: These cannot be currency.EMPTYFORMAT, needs diff addr.
```

Comment:
This comment isn't very helpful as it doesn't explain why. I set these to EMPTYFORMAT and the test still passes